docs/README.md

```diff
@@ -0,0 +1,46 @@
+# Bash-it
+
+[![Build Status](https://travis-ci.com/Bash-it/bash-it.svg?branch=master)](https://travis-ci.com/Bash-it/bash-it)
+[![Join the chat at https://gitter.im/Bash-it/bash-it](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Bash-it/bash-it?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
I realize not many people use irc, but could the Freenode channel be included as well?

https://webchat.freenode.net/?channel=#bash-it

Reply:
This is a good idea, but I think it belongs to a separate PR

I will create an issue to address this :smile: